modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -564,79 +637,48 @@ sudo -u www-data occ user:expire-password
 Arguments:
 
 [width="100%",cols="18%,82%",]
-|=======================================================================
+|===
 |`expiredate` |The date and time when a password expires, e.g.
 ``2019-01-01 14:00:00 CET'' or -1 days
-|=======================================================================
+|===
 
 Options:
 
 [width="100%",cols="18%,82%",]
-|=======================================================================
-|`-a, --all` | Will add password expiry to all known users. uid and group option are discarded if the option is provided by user
-
-|`-u, --uid` | The user’s uid is used. This option can be used as –uid `Alice` –uid Bob"
-
-|`-g, --group` | Add password expiry to user(s) under group(s). This option can be used as –group ``foo'' –group ``bar'' to add expiry passwords for users in group foo and bar. If uid option (eg: –uid ``user1'') is passed with group, then uid will also be processed
-|=======================================================================
-
-[[reports]]
-Reports
--------
-
-If you’re working with ownCloud support and need to send them a configuration summary, you can generate it using the `configreport:generate` command. 
-This command generates the same JSON-based report as the Admin Config Report, which you can access under `admin -> Settings -> Admin -> Help & Tips -> Download ownCloud config report`.
-
-From the command-line in the root directory of your ownCloud installation, run it as your webserver user as follows, (assuming your webserver user is `www-data`):
-
-....
-sudo -u www-data occ configreport:generate
-....
-
-This generates the report and send it to `STDOUT`. 
-You can optionally pipe the output to a file and then attach it to an email to ownCloud support, by running the following command:
-
-....
-sudo -u www-data occ configreport:generate > generated-config-report.txt
-....
-
-Alternatively, you could generate the report and email it all in one command, by running:
-
-....
-sudo -u www-data occ configreport:generate | mail -s "configuration report" \ 
-    -r <the email address to send from> \
-    support@owncloud.com
-....
-
-[NOTE]
-====
-These commands are not available in
-single-user (maintenance) mode <maintenance_commands_label>.
-====
+|===
+|`-a, --all`   | Will add password expiry to all known users.
```

Comment:
That's a nice touch, formatting the raw text like that. It should make it that much easier to maintain. Thanks.